### ISSUE TITLE
False positive of incompatible override

### ISSUE THREAD

**Issue by Azureblade3808** [microsoft/pylance-release#2430]
-   Language Server version: 2022.2.5-pre-1
-   OS and version: Windows 7
-   Python version (& distribution if applicable, e.g. Anaconda): 3.8.12 Anaconda

## Sample code

```python
from __future__ import annotations

from typing import Any

class Base:
    def f(self, *, kwarg0: Any) -> None:
        ...

class Derived(Base):
    def f(self, arg0: Any = ..., *, kwarg0: Any) -> None:
        ...
```

## Expected behavior

No error/warning.

## Actual behavior

![image](https://user-images.githubusercontent.com/15389744/156486143-fdd735ed-38d7-4f44-b498-b88aa8c6db0f.png)

**Comment by erictraut** [comment]
Thanks for the bug report. I've created a tracking bug in the pyright repo.

**Comment by erictraut** [comment]
This will be fixed in the next release.

**Comment by bschnurr** [comment]
This issue has been fixed in version 2022.3.1, which we've just released. You can find the changelog here: [CHANGELOG.md](https://github.com/microsoft/pylance-release/blob/main/CHANGELOG.md#202231-9-March-2022)
